poc/guest/src/main.rs

```diff
@@ -16,9 +10,39 @@ fn panic(_info: &core::panic::PanicInfo) -> ! {
 
 #[polkavm_derive::polkavm_import]
 extern "C" {
-    fn host_call(ptr: u32) -> u32;
+    fn host_call_impl(arg_ptr: u32, out_ptr: u32);
+}
+
+fn sbrk(size: usize) -> *mut u8 {
```

Comment:
just `use polkavm_derive::sbrk` and no need this

Reply:
I was considering to make it more type-safe.

Reply:
what is the type of polkavm_derive::sbrk? looks like it is the same thing?